test/pub_test.dart

```diff
@@ -149,7 +139,7 @@ dev_dependencies:
 PSDependency findDependency(PSDependencyList deps, {String name}) =>
     deps.firstWhere((dep) => dep.name.text == name, orElse: () => null);
 
-testDepListContains(
+void testDepListContains(
```

Comment:
meh.  Still not convinced.  For API I'm with you.  For test code that isn't exported what is it buying me?